docs/content/envoy-performance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Performance
+kind: envoy
+weight: 100
+---
+
+This page provides some performance benchmarks that give an idea of the overhead of using the OPA-Envoy plugin.
+
+### Test Setup
+
+The setup uses the same example Go application that's described in the [Quick Start](#quick-start) section above. Below
+are some more details about the setup:
+
+* Platform: Minikube
+* Kubernetes Version: 1.18.6
+* Envoy Version: 1.10.0
```

Comment:
Hm! It hadn't occurred to me when reviewing this over in the other repo, but why 1.10.0? The latest is 1.17.0 🤔

Reply:
👍 I think we must have missed this in the original PR. We should definitely publish w/ the most recent stable version. v1.10.0 is almost 2 years old.

Reply:
Updated the numbers and graphs based on results from Envoy v1.17.0.